src/sinks/azure_monitor_logs/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod config;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Needs module level doc comment

Reply:
Yes, thank you. I forgot about that 😅